packages/contracts-bedrock/deploy-config/hardhat.json

```
@@ -5,7 +5,8 @@
   "l1StartingBlockTag": "earliest",
   "l1ChainID": 900,
   "l2ChainID": 901,
-  "l2BlockTime": 2,
+  "l2BlockTime": 1,
+  "l1BlockTime": 2,
```

Was this intentional, or should this be
```suggestion
  "l2BlockTime": 2,
  "l1BlockTime": 1,
```

---

Also looks like everywhere else is set to `12`?